src/Server.cpp

```diff
@@ -35,6 +38,36 @@ RouteCfg* find_route(const HttpRequest& req, std::vector<RouteCfg>& routes)
     return route_match.second;
 }
 
+/*
+*   @check_request_method:
+*    Checks if the requested method is accepted by the route
+*/
+bool    check_request_method(RouteCfg* route, const std::string method) {
+    for (size_t i = 0; i < route->accepted_methods.size(); i++)
+        if (method == route->accepted_methods[i])
+            return true;
+    return false;
```

Comment:
A method already exists for finding something in a container (or any other iterrable): https://en.cppreference.com/w/cpp/algorithm/find. I would recommend using it.

Reply:
Sure